Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2158,10 +2158,21 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin()
     setPipeline(gst_element_factory_make("playbin", "play"));
     setStreamVolumeElement(GST_STREAM_VOLUME(m_pipeline.get()));
 
+#if ENABLE(TEXT_SINK)
     unsigned flagText = getGstPlayFlag("text");
+#else
+    unsigned flagText = 0x0;
+#endif
+    
     unsigned flagAudio = getGstPlayFlag("audio");
     unsigned flagVideo = getGstPlayFlag("video");
+    
+#if ENABLE(NATIVE_VIDEO)    
```

Comment:
we should define these options to make them available for the build 
https://github.com/Metrological/WebKitForWayland/blob/master/Source/cmake/OptionsWPE.cmake